src/main/java/seedu/dengue/ui/HelpWindow.java

```diff
@@ -16,20 +16,29 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL = "https://ay2223s2-cs2103-w17-2.github.io/tp/UserGuide.html";
-    public static final String HELP_MESSAGE = "The following are the available commands for this application:\n"
-            + "add, list, edit, find, delete, clear, help, exit\n"
-            + "Formatting:\n"
-            + "add: add n/PATIENT_NAME p/POSTAL_CODE d/DATE a/AGE [v/DENGUE_VARIANT]...\n"
-            + "list: list\n"
-            + "edit: edit INDEX [n/NAME] [p/POSTAL] [d/DATE] [a/AGE] [v/DENGUE_VARIANT]...\n"
-            + "find: find KEYWORD [MORE_KEYWORDS]\n"
-            + "delete: delete INDEX\n"
-            + "clear: clear\n"
-            + "help: help\n"
-            + "exit: exit\n"
-            + "If you want more information, you can refer to the user guide:\n"
-            + USERGUIDE_URL;
-
+    public static final String HELP_MESSAGE = "If you want more information, you can refer to the user guide:\n"
+            + USERGUIDE_URL + "\n"
+            + "\nFor a brief summary, the following are the available commands for this application:\n"
+            + "add, list, edit, find, delete, clear, undo, redo, sort, overview, import, export, help, exit\n"
+            + "\nFormatting:\n"
+            + "\nadd: add n/PATIENT_NAME p/POSTAL_CODE d/DATE a/AGE [v/DENGUE_VARIANT]...\n"
+            + "\nlist: list\n"
+            + "\nedit: edit INDEX [n/NAME] [p/POSTAL] [d/DATE] [a/AGE] [v/DENGUE_VARIANT]...\n"
+            + "\nfind: find [n/NAME] [p/POSTAL] [d/DATE] [a/AGE] [v/DENGUE_VARIANT]...\n"
+            + "or find [n/NAME] [p/POSTAL] [sd/STARTDATE] [ed/ENDDATE] [a/AGE] [v/DENGUE_VARIANT]...\n"
+            + "or find [n/NAME] [p/POSTAL] [d/DATE] [sa/STARTAGE] [ea/ENDAGE] [v/DENGUE_VARIANT]...\n"
+            + "\ndelete INDEX…\n"
+            + "or delete [d/DATE]\n"
+            + "or delete [sd/STARTDATE] [ed/ENDDATE]\n"
+            + "\nclear: clear\n"
+            + "\nundo: undo [INTEGER]\n"
+            + "\nredo: redo [INTEGER]\n"
+            + "\nsort: sort [n/] [a/] [d/]\n"
```

Comment:
Need to find a way to say at least one of them

Reply:
```suggestion
            + "\nsort: sort n/ or a/ or d/\n"
```

Reply:
Done with { | }. Will need to add to UG later.